tests/endtoend_tests/test_examples.py

```diff
@@ -103,6 +103,7 @@ class SampleRequest:
     output_contains_pattern: str | list[str] | None = None
     output_contains_array: npt.ArrayLike | None = None
     output_format: str = "json+base64"
+    extra_cli_args: tuple[str] = ()
```

Comment:
Introduced this to allow passing custom `--eps` or input / output combinations to `check-gradients`, but didn't end up using it. Could remove it if preferred.

Reply:
Let's delete this 🙂